client/app/changelog/changelog.html

```diff
@@ -1,9 +1,9 @@
 <div data-ng-controller="ChangelogCtrl as Changelog">
     <div class="modal-header"></div>
     <div class="modal-body">
-        <oui-loader data-ng-if="!Changelog.content"
-                    data-size="m">
```

Comment:
(optionnal) We can remove the attribute `data-size="m"` on all `<oui-spinner>`. It's default value of the attribute (see http://master.ui-kit.ovh/#!/oui-angular/spinner)